src/Symfony/Component/HttpFoundation/UriSigner.php

```diff
@@ -22,11 +20,14 @@ class UriSigner
     private string $parameter;
 
     /**
-     * @param string $secret    A secret
      * @param string $parameter Query string parameter to use
      */
     public function __construct(#[\SensitiveParameter] string $secret, string $parameter = '_hash')
     {
+        if (!$secret) {
```

Comment:
Do we consider `'0'` as an empty secret?

Reply:
yes, it's equally weak